### ISSUE TITLE
PV doesn't get created, PVC stuck at pending (ARM k3s)

### ISSUE THREAD

**Issue by FlexibleToast** [kadalu/kadalu#283]
Kadalu successfully deployed, storage pod successfully deployed, but when I created a PVC it is stuck at "Pending"
```
---
apiVersion: kadalu-operator.storage/v1alpha1
kind: KadaluStorage
metadata:
 # This will be used as name of PV Hosting Volume
  name: storage1
spec:
  type: Replica1
  storage:
    - node: pikubed-m0
      device: /dev/sda
```

```
---
kind: PersistentVolumeClaim
apiVersion: v1
metadata:
  name: pv1
spec:
  storageClassName: kadalu.replica1
  accessModes:
    - ReadWriteOnce # You can also provide 'ReadWriteMany' here
  resources:
    requests:
      storage: 1Gi
```

Attaching logs of the provisioner:
[csi-provisioner.txt](https://github.com/kadalu/kadalu/files/5010462/csi-provisioner.txt)
[csi-attacher.txt](https://github.com/kadalu/kadalu/files/5010463/csi-attacher.txt)
[kadalu-provisioner.txt](https://github.com/kadalu/kadalu/files/5010464/kadalu-provisioner.txt)
Kadalu-logging container had nothing in the kubectl logs output.

**Comment by amarts** [comment]
With #262 and #284 merged, `master` tag should be good to be deployed on ARM and it should work. Can you please check and confirm when you get time?

**Comment by FlexibleToast** [comment]
I just realized I never got back with this. I just tested Kadalu again today. This is using 4x Raspberry Pi 4s, Ubuntu 20.04, k3s 1.19.4 with their embedded etcd. Everything installs correctly. However, things tend get hung which results in this issue. This happens in two places. When I try to add a storage pool initially nothing happened. If I delete the operator pod when it gets recreated then the storage pools create without issue. It's as if the operator gets hung and needs to be reset. Samething occured with the provisioner. I created the sample-pvc.yml and after I applied it the pvc got created, but the pv wasn't created until I deleted the provisioner pod.
